.github/workflows/support-window-issue.yml

```diff
@@ -0,0 +1,36 @@
+name: Support Windows Update Issue Creater
```

Comment:
The name of the workflow is quite lengthy and will be truncated in UI. Why not go with `Support Windows Issue Creator`(notice the typo in creator too)
Also the workflow file name doesn't convey the purpose, it can be anything like updating, deleting, modifying or creating the issue. Why not make it explicit by naming it  `support-window-issue-creator.yml`
You can also take a look at:
https://openedx.atlassian.net/wiki/spaces/AC/pages/3548282937/GitHub+Actions+best+practices

Reply:
updated as per the suggestions.